### PR TITLE
Clean time

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

**Review by alexkalmuk — COMMENTED** [review]
ntpdate + date work on arm/qemu, but not on x86/qemu:

```
>ntpdate 54.36.110.36
>tish
root@embox:/#date
1970-01-01 00:00:02
root@embox:/#
```

It's ok?

File: templates/arm/qemu/mods.conf [hunk 1]
```diff
@@ -114,6 +114,7 @@ configuration conf {
 	include embox.cmd.net.tftp
 	include embox.cmd.net.snmpd
 	include embox.cmd.net.ntpdate
+	include embox.cmd.net.ntpd
```
Comment (alexkalmuk) [inline templates/arm/qemu/mods.conf:117]: It's working? Or added just to make sure t compiled?

File: src/compat/posix/util/time.c [hunk 2]
```diff
@@ -9,27 +9,55 @@
...
-	ktime_get_timespec(ts);
+	switch(clk_id) {
+		case CLOCK_REALTIME: {
+			getnsofday(ts, NULL);
+			break;
+		}
+		case CLOCK_MONOTONIC: {
+			ktime_get_timespec(ts);
+			break;
+		}
```
Comment (alexkalmuk) [inline src/compat/posix/util/time.c:32]: You do not need `{` in cases. Why you use them? It's convenient or what? (as for me - not)

**Review by alexkalmuk — APPROVED** [review]
LGTM

**Comment by anton-bondarev** [comment]
> ntpdate + date work on arm/qemu, but not on x86/qemu:
> 
> ```
> >ntpdate 54.36.110.36
> >tish
> root@embox:/#date
> 1970-01-01 00:00:02
> root@embox:/#
> ```
> 
> It's ok?
No, it's a troube. Thanks! I described it in #2257

No, you are right, thanks. I described the trouble in issue #2257 2257

File: templates/arm/qemu/mods.conf (hunk 1 above)
Comment (anton-bondarev) [reply templates/arm/qemu/mods.conf:117]: It's only for the checking compilation

File: src/compat/posix/util/time.c (hunk 2 above)
Comment (anton-bondarev) [reply src/compat/posix/util/time.c]: Thanks. Done

**Pull request merged**